appveyor.yml

```diff
@@ -29,6 +29,6 @@ test_script:
 
 environment:
   matrix:
-    - ruby_version: '200'
```

Comment:
Originated in rspec/rspec-dev#113, which passed the build here and got merged but failed on rspec-core (see rspec/rspec-core#1861 and rspec/rspec-core#1868).  Not a big deal right now; once the rspec-core issue is dealt with we'll rebase the rspec-dev PR and push out updates to all the build scripts.  So ignore for now.